src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -100,4 +100,50 @@ private void waitForCredentialVisible(final String credUsername) {
         // Select the new credentials. Control.selectDropdownMenu seems to be YUI-only.
         credentialsId.select(credUsername);
     }
+
+    public void setSshHostKeyVerificationStrategy(Class<? extends SshHostKeyVerificationStrategy> type) {
+        try {
+            SshHostKeyVerificationStrategy strategy = type.newInstance();
+            hostKeyVerificationStrategy.select(strategy.id());
+        } catch (InstantiationException | IllegalAccessException e) {
+            throw new IllegalArgumentException(type.getName() + " is missing a default constructor.", e);
+        }
+
+    }
+
+    public abstract static class SshHostKeyVerificationStrategy {
+        public abstract String id();
```

Comment:
Is this really necessary? Normally `Describable`s in a list are selected by `Descriptor.displayName` or `.id`. Relying on a numeric position is pretty fragile. Perhaps you overlooked the typical methods for picking an item by label?